server/routes/place.py

```diff
@@ -47,14 +47,17 @@
 @bp.route('/<path:place_dcid>', strict_slashes=False)
 def place(place_dcid=None):
     dcid = flask.request.args.get('dcid', None)
-    topic = flask.request.args.get('topic', None)
+    category = flask.request.args.get('category', None)
+    if not category:
+        # Used to use topic as the url param.
+        category = flask.request.args.get('topic', None)
```

Comment:
I think we should also issue a redirect, if not from search, from &topic to &category so we don't increase the number of url's in the search index (and for consistency)

Reply:
Always redirect to ?category now.